src/main/java/com/traveltime/sdk/TravelTimeSDK.java

```diff
@@ -142,10 +143,34 @@
         return getFuture(sendProtoAsyncBatched(request, batchSizeHint));
     }
 
+    public Either<TravelTimeError, TimeFilterFastProtoResponse> sendProtoBatchedCount(
```

Comment:
The names got pretty bad, I'm considering removing the methods that take a size hint as an argument.